### DISCUSSION TITLE
API EndPoint and RPC Setup

### DISCUSSION THREAD

**Discussion by jsim77** [filecoin-project/lotus#8328] [Build and Sync]
Hi,

In my Mac terminal the Lotus Version is 1.14.1+mainnet+git. Sorry, I need help in getting the API using Mac and yes, I do have Infura's endpoint but how to connect it? I did follow the Docs, can I run this via Postman? What about the RPC setup? Sorry again as I'm not a tech-savvy in this.

**Comment by TippyFlitsUK** [comment]
Hey @jsim77 

Can you provide a little more information with regards to your setup and what you are trying to achieve?

Many thanks!
